### ISSUE TITLE
ValueError: Math domain error running MDA with MTO=90000

### ISSUE THREAD

**Issue by jpruscio** [fast-aircraft-design/FAST-OAD#119]
I changed only the MTO (maximum thrust at SL) value from the original input xml file to 90000 N, I ran MDA, and I obitained the error attached

I downloaded and installed again FAST-OAD today from the link indicated in the readme:
https://github.com/fast-aircraft-design/FAST-OAD/archive/master.zip

[fast oad error 90000.zip](https://github.com/fast-aircraft-design/FAST-OAD/files/4199346/fast.oad.error.90000.zip)

Thanks!

**Comment by ScottDelbecq** [comment]
Hi, 
I have just checked this issue for the beta release. I cannot reproduce the error anymore. The MDA process has been restructured since and might have solved this issue. @christophe-david or @jpruscio can you test and confirm?

**Comment by christophe-david** [comment]
You might have a look at PR #126, that I created to secure the branch where this failure case was reproduced.
I had once replayed this case after rebasing the branch after the restructuration of the process and got the same result as you.

After that, I experimented a bit and found that with a different module order, the crash could happen again. Therefore, I prefer to keep this issue alive, so we can investigate deeper in the future to ensure more robustness.
